pallets/gear/src/benchmarking/code.rs

```diff
@@ -263,65 +262,6 @@ impl<T: Config> WasmModule<T>
 where
     T: Config,
 {
-    /// Uses the supplied wasm module and instruments it when requested.
-    pub fn instrumented(code: &[u8], inject_gas: bool, inject_stack: bool) -> Self {
```

Comment:
I mean that bechmarks related code. Is it not need anymore?

Reply:
Yes, it's completely unused